kitsune/community/utils.py

```diff
@@ -95,40 +136,27 @@ def _apply_filters(query, start, end, locale=None, product=None):
 
 
 def _get_creator_counts(query, count, page):
-    """Get the list of top contributors with the contribution count."""
-    creator_counts = query.facet_counts()['creator_id']['terms']
-
-    total = len(creator_counts)
-
-    # Pagination
-    creator_counts = creator_counts[((page - 1) * count):(page * count)]
-
-    # Grab all the users from the user index in ES.
-    user_ids = [x['term'] for x in creator_counts]
-    results = (
-        UserMappingType
-        .search()
-        .filter(id__in=user_ids)
-        .values_dict('id', 'username', 'display_name', 'avatar',
-                     'twitter_usernames', 'last_contribution_date'))[:count]
-    results = UserMappingType.reshape(results)
-
-    # Calculate days since last activity and
-    # create a {<user_id>: <user>,...} dict for convenience.
-    user_lookup = {}
-    for r in results:
-        lcd = r.get('last_contribution_date', None)
-        if lcd:
-            delta = datetime.now() - lcd
-            r['days_since_last_activity'] = delta.days
-        else:
-            r['days_since_last_activity'] = None
-
-        user_lookup[r['id']] = r
-
-    # Add the user to each dict in the creator_counts array.
-    for item in creator_counts:
-        item['user'] = user_lookup.get(item['term'], None)
-
-    return ([item for item in creator_counts if item['user'] is not None],
-            total)
+    total = query.count()
+    results = []
+    now = datetime.now()
+    for user in query[((page - 1) * count):(page * count)]:
+        last_contribution_date = user.profile.last_contribution_date
+        days_since_last_activity = None
+        if last_contribution_date:
+            days_since_last_activity = now - last_contribution_date
+
+        data = {
+            'count': user.query_count,
+            'term': user.id,
+            'user': {
+                'username': user.username,
```

Comment:
here should be `user.id` also
https://github.com/mozilla/kitsune/blob/master/kitsune/community/jinja2/community/index.html#L163